editor/js/Loader.js

```diff
@@ -93,7 +93,14 @@ function Loader( editor ) {
 					loader.setLibraryPath( '../examples/jsm/libs/rhino3dm/' );
 					loader.parse( contents, function ( object ) {
 
+						object.name = filename;
+
 						editor.execute( new AddObjectCommand( editor, object ) );
+						console.log( object )
```

Comment:
Do you mind removing this line. We usually do not log entire objects in the browser console.

Reply:
yep, oversight, fixed in next commit.